test/test_testmon.py

```diff
@@ -293,7 +291,7 @@ def test_2():
         tf.setmtime(1800000000)
         try:
             testdir.runpytest("--testmon", )
-        except:
```

Comment:
Why removing the KeyboardInterrupt?

Reply:
It uses it explicitly now.. (I had just `Exception` there before, which does not cover `KeyboardInterrupt`)